src/message/header/date.rs

```diff
@@ -14,7 +17,10 @@ pub struct Date(HttpDate);
 impl Date {
     /// Build a `Date` from [`SystemTime`]
     pub fn new(st: SystemTime) -> Self {
```

Comment:
Sorry I missed this in the previous review - The above changes with the import statements made it so `Date::new` takes `web_time::SystemTime`, which makes `web` a non-additive feature. Could this take `std::time::SystemTime` instead?

This also applies to the `impl From<Date> for SystemTime` and `impl From<SystemTime> for Date` impls in this same file.

Reply:
Let me try it on wasm env. and get back on this. From my experience, the part of std Time that panics is the now() function. Unfortunately, std::time compiles to wasm32-unknown-unknown, so you can only find out what works/not afterwords.

Reply:
Could we have the function take std `SystemTime`, but `now` always uses `web_time::SystemTime` and then converts to std `SystemTime`?